scripts/deploy.sh

```diff
@@ -25,7 +25,11 @@ ls $DEPLOY_CONF_DIR
 #update the repository
 cd $DEPLOY_SERVICE_DIR
 git config --global user.name "osscameroon-bot"
-git pull --rebase https://$GITHUB_TOKEN@github.com/osscameroon/deployments main
+
+#get repo url takes a `  Fetch URL: git@github.com:elhmn/infra.git`
+#and returns `@github.com/elhmn/infra.git`
+REPO_PATH=$(git remote show origin | grep Fetch | cut -d ":" -f 2- | tr -d " " | tr -s ":" "/" | sed -s "s/^git//g")
```

Comment:
```suggestion
REPO_PATH=$(grep Fetch <<< $(git remote show origin) | sed -E 's/^.*?(git@|https:\/\/)([^/]+)\/(.*)$/@\2:\3/')
```

- less pipes and commands
- this works with git@github:... like URLs and also https://github... like URLs

```
$ url="Fetch URL: https://github.com/x/y.git"
$ grep Fetch <<< $(echo $url) | sed -E 's/^.*?(git@|https:\/\/)([^/]+)\/(.*)$/@\2:\3/'
@github.com:x/y.git

$ grep Fetch <<< $(echo $url) | sed -E 's/^.*?(git@|https:\/\/)([^/]+)\/(.*)$/@\2:\3/'
@github.com:x:y.git
```